website/docs/components/badge/index.md

```diff
@@ -9,8 +9,6 @@ related: ['components/badge-count','components/tag']
 previewImage: assets/illustrations/components/badge.jpg
 navigation:
   keywords: ['chip', 'pill', 'tag', 'label']
-status:
-  updated: 4.7.0
```

Comment:
more a meta conversation to have outside of this PR, but removing this already feels pretty fast to me? this was only released a week ago, so it feels like a very small window for the update notification to be present/seen by consumers.

Reply:
I agree, but then when I looked at the side nav with all that "updated" and "added" it felt overwhelming and a lot of noise, hard to understand what was really new. 

Anyway, I've removed the commit: as soon as the preview website is regenerated have a look and let me know what you think

/cc @heatherlarsen @majedelass your feedback would be appreciated

Reply:
Here's the version with the `4.7.0` and `4.8.0` updates: https://hds-website-git-appheader-version-history-hashicorp.vercel.app/components

Reply:
hot take (and I know Cristiano will prepare a more elaborated proposal for this): I would be tempted for us to keep them not based on time, but based on releases:
 - "deprecated": should stay until removal (ideally this would be the next major release)
 - "added": considering we don't add as many components (at least not at this stage) this could stay as well until the next major, or at least a couple of minor releases (say 3)
 - "updated": a couple of minor releases (3 or so sounds reasonable to me, and could be adjusted over time depending on how often we release)

Reply:
OK, since the consensus seems to be around keeping the bagdes for a bit longer (and considering that removing them is a 5 minutes job) I'm going to merge this PR as is, keeping also the `4.7.0` in place